### ISSUE TITLE
PAYARA-4104 Disabling Hazelcast for 5.193 Server causes startup failure

### ISSUE THREAD

**Issue by mwnorman** [payara/Payara#4186]
# Description #

Server fails to start when hazelcast is disabled.

## Current Outcome
On macOS, the port number for (VNC) screen-sharing is 5900 which is used by hazelcast.
Disabling it via:
`asadmin set-hazelcast-configuration --enabled=false`
results in a NPE:

```
MultiException stack 1 of 1
MultiException stack 1 of 2
java.lang.NullPointerException
        at fish.payara.monitoring.store.InMemoryMonitoringDataRepository.init(InMemoryMonitoringDataRepository.java:116)
        
```
## Steps to reproduce
Starting with un-modified "domain1" config ...
Temporarily disable Screen Sharing
`asadmin start-domain`
`asadmin set-hazelcast-configuration --enabled=false`
`asadmin stop-domain`
Re-enable Screen Sharing
`asadmin start-domain`

## Environment ##

- **Payara Version**: 5.193
- **Edition**: Server (Full)
- **JDK Version**:
`openjdk version "1.8.0_222"`
`OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_222-b10)`
`OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.222-b10, mixed mode)`
- **Operating System**: macOS 10.14.6

**Comment by mwnorman** [comment]
work-around:
`<hazelcast-runtime-configuration start-port="59000"></hazelcast-runtime-configuration>`

since it is nigh-impossible to change the Screen Sharing port on the Mac

**Comment by jbee** [comment]
Thanks @mwnorman for fast report and workaround. We unfortunately missed that new monitoring console at the moment requires hazelcast turned on. 

Another workaround I want to verify is to remove `monitoring-console-core.jar` from `glassfish/modules` folder.

**Comment by jbee** [comment]
Tracked by internal issue PAYARA-4104.

**Comment by jbee** [comment]
Fixed by #4188 . As a workaround remove the `monitoring-console-core.jar` from `glassfish/modules`.
